teams/wg-macros.toml

```
@@ -26,3 +26,6 @@ zulip-stream = "wg-macros"
 
 [[zulip-groups]]
 name = "wg-macros"
+
+[permissions]
+perf = true
```

you also need `bors.rust.try = true` in order to `@ bors try`
```suggestion
perf = true
bors.rust.try = true
```

---

Thanks!